src/emqx_cm.erl

```diff
@@ -141,6 +148,12 @@ unregister_channel(ClientId) when is_binary(ClientId) ->
 do_unregister_channel(Chan) ->
     ok = emqx_cm_registry:unregister_channel(Chan),
     true = ets:delete(?CHAN_CONN_TAB, Chan),
+    case ets:lookup(?CHAN_INFO_TAB, Chan) of
```

Comment:
Now I'm not sure if refreshing the counter will fix or cause discrepancy
because this unregister is NOT done by the `?CM` process, but `emqx_pool` members,
refreshing the counter with a `ets:select` in `?CM` may race with the deletes.

Reply:
Maybe if we check the connection states tied to those channels and decrementing inside `emqx_cm` (during the `handle_info` for `DOWN` messages) prior to handing the `emqx_cm:clean_down/0` tasks to the `emqx_pool` workers. :thinking:

Reply:
Maybe this refresh is not necessary because even if the pid is killed, there will be a DOWN message to trigger the decrement.

could you remove the refresh code for now.
and maybe add it in another PR if it's necessary after all.